CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+## Development workflow
+
+To get started with the project, run `yarn` in the root directory to install the required dependencies:
+
+```sh
+yarn
+```
+
+We use [storybook](https://storybook.js.org/docs) for components development. To run storybook execute
+
+```sh
+yarn dev
+```
+
+We use [Chromatic](https://www.chromatic.com/docs/) for visual testing and UI regression.
+
+Our storybook is published to Chromatic and accessible by following: [https://storybook.lw.iog.io/](https://storybook.lw.iog.io/)
+
+### Commit message convention
+
+We follow the [conventional commits specification](https://www.conventionalcommits.org/en) for our commit messages:
+
+- `fix`: bug fixes, e.g. fix crash due to deprecated method.
+- `feat`: new features, e.g. add new method to the module.
+- `refactor`: code refactor, e.g. migrate from class components to hooks.
+- `docs`: changes into documentation, e.g. add usage example for the module..
+- `test`: adding or updating tests, e.g. add integration tests using detox.
+- `chore`: tooling changes, e.g. change CI config.
+
+Our pre-commit hooks verify that your commit message matches this format when committing.
+
+### Linting and tests
+
+[ESLint](https://eslint.org/), [Prettier](https://prettier.io/), [TypeScript](https://www.typescriptlang.org/)
```

Comment:
Is it worth mentioning here and line below, twice? 🤔

Reply:
removed